framework/include/tinyalsa/tinyalsa.h

```diff
@@ -138,7 +138,8 @@ extern "C" {
  */
 #define	PCM_STATE_DISCONNECTED 0x08
 
-/** Audio sample format of a PCM.
+/** 
+ * @brief Audio sample format of a PCM.
  * The first letter specifiers whether the sample is signed or unsigned.
```

Comment:
I think you had better add details in here.

Reply:
yes i added "@details"